src/bin/vip-wp.js

```diff
@@ -192,6 +194,8 @@ const launchCommandAndGetStreams = async ( { guid, inputToken, offset = 0 } ) =>
 
 		rollbar.error( err );
 		console.log( err );
+
+		process.exit( 1 );
```

Comment:
Could we maybe add a threshold for the rate limit errors?  like after 3 attempts, the then `process.exit()`s